runtime/doc/eval.txt

```diff
@@ -4034,6 +4035,19 @@ findfile({name} [, {path} [, {count}]])				*findfile()*
 <		Searches from the directory of the current file upwards until
 		it finds the file "tags.vim".
 
+flatten({list} [, {maxdepth}])					*flatten()*
+		Flatten {list} to depth {maxdepth}.  {maxdepth} means how
+		deeply it makes flatten the list.  the depth is 1 when
+		{maxdepth} is omitted.  {list} is not modified when
+		{maxdepth} is 0.
+		Example: >
+			:echo flatten([1, [2], 3])
+<			[1, 2, 3] >
+			:echo flatten([[[1], [2], [3]]], 1)
+<			[[1], [2], [3]]
+							        *E964*
+		{maxdepth} must be positive number.
```

Comment:
Since zero is explicitly allowed (why?), this should say "must be a non-negative Number."

Reply:
This description is right because the given list is NOT modified when the `maxdepth` is 0.

It can deal with when a calculation result of `maxdepth` becomes 0.